manifests/mod/passenger.pp

```diff
@@ -56,17 +56,32 @@
   }
 
   if $::osfamily == 'RedHat' and $manage_repo {
-    yumrepo { 'passenger':
-      ensure        => 'present',
-      baseurl       => 'https://oss-binaries.phusionpassenger.com/yum/passenger/el/$releasever/$basearch',
-      descr         => 'passenger',
-      enabled       => '1',
-      gpgcheck      => '0',
-      gpgkey        => 'https://packagecloud.io/gpg.key',
-      repo_gpgcheck => '1',
-      sslcacert     => '/etc/pki/tls/certs/ca-bundle.crt',
-      sslverify     => '1',
-      before        => Apache::Mod['passenger'],
+    if $::operatingsystem == 'Amazon' {
+      yumrepo { 'passenger':
+        ensure        => 'present',
+        baseurl       => 'https://oss-binaries.phusionpassenger.com/yum/passenger/el/6Server/$basearch',
```

Comment:
If I am not mistaken, isn't the only change in this PR hardcoding 6Server rather than using $releasever on Amazon? Would it be tidier to move the base_url out to a separate variable, rather than duplicate the entire yumrepo resource? e.g. 
`if $::operatingsystem == 'Amazon' {`
`  $passenger_base_url = 'https://oss-binaries.phusionpassenger.com/yum/passenger/el/6Server/$basearch'`
`} else {`
`  $passenger_base_url = 'https://oss-binaries.phusionpassenger.com/yum/passenger/el/$releasever/$basearch'`
`}`